Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2539,6 +2539,15 @@ - (BOOL)_canTogglePictureInPicture
 #endif
 }
 
+- (BOOL)_canToggleInWindow
+{
+#if HAVE(TOUCH_BAR)
+    return _impl->canTogglePictureInPicture();
+#else
+    return NO;
+#endif
+}
+
```

Comment:
This is fine for now, but I think we will eventually want a non HAVE_TOUCH_BAR path.